packages/api/src/resolvers/queries/worker/workQueueCount.ts

```diff
@@ -0,0 +1,33 @@
+import { log } from 'meteor/unchained:logger';
+import { Root, Context } from '@unchainedshop/types/api';
+import { WorkStatus } from '@unchainedshop/types/worker';
+
+export default async function workQueueCount(
+  root: Root,
+  {
+    status,
+    selectTypes,
+    created,
+    queryString,
+  }: {
+    queryString?: string;
+    status?: Array<WorkStatus>;
+    selectTypes?: Array<string>;
```

Comment:
selectTypes is deprecated, the new workQueueCount should only support the non deprecated "types"